README.md

```diff
@@ -85,49 +104,6 @@ your personal usage, some of these will also need to be installed. Refer to `:ch
 [winzip]: https://www.winzip.com/
 [winrar]: https://www.win-rar.com/
 
-# Installation
-
-## [Packer](https://github.com/wbthomason/packer.nvim)
-
-```lua
-use {
-    "mason-org/mason.nvim"
-}
-```
-
-## [lazy.nvim](https://github.com/folke/lazy.nvim)
-
-```lua
-{
-    "mason-org/mason.nvim"
-}
-```
-
-## [vim-plug](https://github.com/junegunn/vim-plug)
-
-```vim
-Plug 'mason-org/mason.nvim'
-```
-
-# Setup
-
-> [`:h mason-quickstart`][help-mason-quickstart]
-
-```lua
-require("mason").setup()
-```
-
-`mason.nvim` is optimized to load as little as possible during setup. Lazy-loading the plugin, or somehow deferring the
-setup, is not recommended.
-
-Refer to the [Configuration](#configuration) section for information about which settings are available.
-
-## Extensions
-
-Refer to the [Wiki](https://github.com/mason-org/mason.nvim/wiki/Extensions) for a list of 3rd party extensions.
-
--   [`mason-lspconfig.nvim`](https://github.com/mason-org/mason-lspconfig.nvim) - recommended for usage with `lspconfig`
-
```

Comment:
This block is completely removed in the rewrite. Is that intentional? (Just checking in case you forgot to paste it after cutting.)

Reply:
Yeah this removal was intentional. `mason-lspconfig.nvim` isn't really as necessary anymore, and I kinda want to remove the idea of a "Mason ecosystem". I see a lot of people being confused about what the Mason plugins does and they end up assuming that either Mason is a requirement for LSP etc. or that `mason-lspconfig.nvim` is a requirement for usage with `nvim-lspconfig`, neither of which is true.